udong-frontend/app/ui/feature/header/Header.tsx

```diff
@@ -40,14 +40,14 @@ export const Header = ({ type, clubId }: HeaderProps) => {
                         src={Logo.src}
                         height={50}
                         width={160}
-                        onClick={() => router.push('/')}
+                        onClick={() => router.push('/').then(()=>router.reload())}
```

Comment:
다른 페이지에서 헤더 클릭하니까 이동된 다음에 한 번 더 reload돼서, 이건 어때?
```suggestion
                        onClick={() => {
                            if(router.pathname === '/' && router.query['tab'] !== 'mydong') {router.reload()}
                            else {router.push('/')}
                        }}
```